### PR TITLE
[View/Compiler] Add Compiler Toolchain View

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit adds compiler toolchain view on ONE view

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,14 @@
       ]
     },
     "views": {
-      "ONE-Views": []
+      "ONE-Views": [
+        {
+          "id": "CompilerView",
```
Comment (hyunsik-yoon) [inline package.json:37]: Which name would be better among `Compiler` and `CompilationEnv`?
I guess we should use one name through entire project. :-)

Comment (YongseopKim) [reply package.json:37]: IHMO, how about `CompilationView`? AFAIU, the `CompilationEnv` could be multiple on and be an env instance itself ONE-vscode.

Comment (hyunsik-yoon) [reply package.json:37]: /cc @Samsung/one-vscode

**Comment by jyoungyun** [comment]
/cc @YongseopKim @hyunsik-yoon 

Please take an another look.

**Review by YongseopKim — APPROVED** [review]
LGTM 👍

**Review by hyunsik-yoon — APPROVED** [review]
LGTM

**Comment by jyoungyun** [comment]
Please merge this PR

**Comment by jyoungyun** [comment]
Thank you /cc @seanshpark  :)

**Pull request merged**